tensorzero-core/src/inference/types/storage.rs

```diff
@@ -75,17 +75,18 @@ impl StorageKind {
     }
 }
 
-#[cfg_attr(test, derive(ts_rs::TS))]
-#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
-#[cfg_attr(test, ts(export))]
+/// Path to a file in an object storage backend.
+/// This is part of the public API for `File`s.
```

Comment:
can you add the use case here? Something like:

"in particular, this is useful for roundtripping unresolved inputs from stored inferences or datapoints without rendering into the inference and other APIs"